parm/atm/obs/config/atms_n20.yaml

```diff
@@ -0,0 +1,371 @@
+obs space:
+  name: atms_n20
+  obsdatain:
+    obsfile: $(OBS_DIR)/$(OBS_PREFIX)atms_n20.$(OBS_DATE).nc4
+  obsdataout:
+    obsfile: $(DIAG_DIR)/diag_atms_n20_$(OBS_DATE).nc4
+  io pool:
+    max pool size: 1
+  simulated variables: [brightness_temperature]
+  channels: &atms_n20_channels 1-22
+get values:
+  interpolation method: $(INTERP_METHOD)
+obs operator:
+  name: CRTM
+  Absorbers: [H2O,O3,CO2]
```

Comment:
For H(x) this is fine, but running with Variational DA, we will need a `linear obs operator` section with `CO2` removed, like the amsua file has. Up to you @CatherineThomas-NOAA if you want to do that now or we can do it later.